kiwi/storage/subformat/template/virtualbox_ovf.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2019 SUSE Linux GmbH.  All rights reserved.
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+
+from string import Template
+
+
+class VirtualboxOvfTemplate(object):
+    """
+    **Generate a OVF file template for a vagrant virtualbox box**
+
+    This class provides a template for virtualbox' ovf configuration file that
+    is embedded inside the vagrant box. The template itself was extracted from
+    a vagrant box that was build via packer and from a script provided by Neal
+    Gompa.
+    """
+
+    def __init__(self):
+        self.OVF_TEMPLATE = """<?xml version="1.0"?>
+<Envelope ovf:version="1.0" xml:lang="en-US" xmlns="http://schemas.dmtf.org/ovf/envelope/1" xmlns:ovf="http://schemas.dmtf.org/ovf/envelope/1" xmlns:rasd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData" xmlns:vssd="http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:vbox="http://www.virtualbox.org/ovf/machine">
+  <References>
+    <File ovf:id="file1" ovf:href="box.vmdk"/>
+  </References>
+  <DiskSection>
+    <Info>List of the virtual disks used in the package</Info>
+    <Disk ovf:capacity="${disk_image_capacity}" ovf:diskId="vmdisk1" ovf:fileRef="file1" ovf:format="http://www.vmware.com/interfaces/specifications/vmdk.html#streamOptimized"/>
+  </DiskSection>
+  <NetworkSection>
+    <Info>Logical networks used in the package</Info>
+    <Network ovf:name="NAT">
+      <Description>Logical network used by this appliance.</Description>
+    </Network>
+  </NetworkSection>
+  <VirtualSystem ovf:id="${vm_name}">
+    <Info>A virtual machine</Info>
+    <OperatingSystemSection ovf:id="${cimos_id}">
+      <Info>The kind of installed guest operating system</Info>
+      <Description>${vm_description}</Description>
+      <!--FIXME: this is probably not required<vbox:OSType ovf:required="false">OpenSUSE_64</vbox:OSType>-->
```

Comment:
For OS type, we should just use generic Other Linux / Other Linux 64. :)

Reply:
I'll change that.